### PR TITLE
plot_qc function added to genrate one figure containing all 5 qc plots

### PR DESCRIPTION
**Pull request by NBMueller**

### CODE REVIEW

File: src/scdef/models/_scdef.py [hunk 1]
```diff
@@ -2040,24 +2054,73 @@ def plot_gini_brd(
...
+        else:
+            return ax
 
+    def plot_likelihood(self, figsize=(4, 4), fontsize=12, ax=None, show=True):
+        if ax is None:
+            fig, ax = plt.subplots(figsize=figsize)
+        ax.plot(np.concatenate(self.elbos)[:])
+        ax.set_yscale("log")
+        ax.set_xlabel("# ELBO", fontsize=fontsize)
+        ax.set_ylabel("Likelihood [log]", fontsize=fontsize)
```
Comment (pedrofale) [inline src/scdef/models/_scdef.py]: Not sure I agree with the nomenclature here. Would you consider renaming to `plot_elbos`? And then `ax.set_xlabel('iteration')`, `ax.set_ylabel('ELBO [log]')`. The ELBO is an approximation to the marginal likelihood, but it's not the marginal likelihood itself

File: src/scdef/models/_scdef.py [hunk 2]
```diff
@@ -2040,24 +2054,73 @@ def plot_gini_brd(
...
         if show:
             plt.show()
+        else:
+            return ax
+
+    def plot_qc(self, figsize=(8, 12), show=True):
+        """Plot QC metrics for scDEF run:
+            top left: log-likelihood vs. # ELBO
```
Comment (pedrofale) [inline src/scdef/models/_scdef.py]: ELBO vs iteration

**Review by pedrofale — CHANGES_REQUESTED** [review]
Great idea to centralize this and it will make life much easier for users. Please reconsider the nomenclature in the `plot_likelihood` function. Otherwise, LGTM! Thanks!

File: src/scdef/models/_scdef.py (hunk 1 above)
Comment (NBMueller) [reply src/scdef/models/_scdef.py]: changed to `Loss [log]` vs. `Epoch`

File: src/scdef/models/_scdef.py (hunk 2 above)
Comment (NBMueller) [reply src/scdef/models/_scdef.py]: changed to `Loss (-1xELBO [log]) vs. Epoch`

**Review by pedrofale — APPROVED** [review]

**Comment by pedrofale** [comment]
This is included in version [0.0.18](https://pypi.org/project/scdef/).

**Pull request merged**